### DISCUSSION TITLE
Smalll arrow on top of the pop up.

### DISCUSSION THREAD

**Discussion by NivRash1251** [hakimio/ngx-ui-tour#149] [Q&A] [answered]
Do we have the option to show a small arrow as shown in the image in this library ?
<img width="350" alt="Screenshot 2023-09-29 at 3 14 33 PM" src="https://github.com/hakimio/ngx-ui-tour/assets/137115214/f14c595c-3131-4340-a810-1d7071cbce65">

**Comment by hakimio** [comment]
Depends on the UI you use:
https://hakimio.github.io/ngx-ui-tour/ng-bootstrap
https://hakimio.github.io/ngx-ui-tour/tui-hint

I might add it also to `md-menu` UI in the next major release as well.

**Reply by NivRash1251** [reply]
Ohh okay.. I am using https://hakimio.github.io/ngx-ui-tour/md-menu
. Anyways Thanks.

**Reply by NiveditaRashmi** [reply]
Any plans to add the arrow in https://hakimio.github.io/ngx-ui-tour/md-menu for angular 16 ?

**Reply by hakimio** [reply]
I have plans to try to implement it for the next major release `v12` for `Angular 17`.

**Comment by hakimio** [comment, marked as answer]
`md-menu` UI now includes the arrow. Check out [v12 release announcement](https://github.com/hakimio/ngx-ui-tour/releases/tag/v12.0.0).
